packages/table/test/common/internal/focusCellUtilsTests.ts

```diff
@@ -1,87 +0,0 @@
-/**
```

Comment:
renamed this file to `focusedCellUtilsTests.ts`, in keeping with my goal to use "focused cell" instead of "focus cell," everywhere (and also because the tested module is `FocusedCellUtils` anyway).